rmgpy/tools/fluxdiagram.py

```diff
@@ -442,7 +442,7 @@ def loadChemkinOutput(outputFile, reactionModel):
 ################################################################################
 
 def createFluxDiagram(inputFile, chemkinFile, speciesDict, savePath=None, speciesPath=None, java=False, settings=None,
-                      chemkinOutput='', centralSpecies=None, diffusionLimited=True):
+                      chemkinOutput='', centralSpecies=None, diffusionLimited=True, checkDuplicates=True):
```

Comment:
Does this tool only work for RMG-Py mechanisms given that it needs an RMG  input file? If so I think the default option here should be `checkDuplicates=False` (as well as for scripts/generateFluxDiagram.py).

@mliu49 will this break the website in any way?

If this works for models other than just RMG ones then we should update the documentation for Generating Flux Diagrams to instruct RMG users to set this flag for speed

Reply:
In theory, one could use a non-RMG Chemkin file and create the corresponding species dictionary and input file and pass it to the function, but it is unlikely that anyone would do that.

Reply:
Gotcha. In this case it might be best to leave the defaults like you have them. The best thing to do would be to update the documentation to make RMG users aware of this flag, but we don't currently do that for the other options, and it would show up if the user looked through the options or executed the help flag. Therefore, I leave this decision to you.